third_party/2and3/decorator.pyi

```diff
@@ -0,0 +1,85 @@
+import sys
+from typing import Any, Callable, Dict, List, NamedTuple, Optional, Pattern, Text, Tuple, TypeVar
+
+_C = TypeVar("_C", bound=Callable[..., Any])
+_Func = TypeVar("_Func", bound=Callable[..., Any])
+
+def get_init(cls): ...
+
+# Duplicated from `stdlib/3/inspect.pyi`
+FullArgSpec = NamedTuple('FullArgSpec', [('args', List[str]),
+                                         ('varargs', Optional[str]),
+                                         ('varkw', Optional[str]),
+                                         ('defaults', tuple),
+                                         ('kwonlyargs', List[str]),
+                                         ('kwonlydefaults', Dict[str, Any]),
+                                         ('annotations', Dict[str, Any]),
+                                         ])
+
+if sys.version_info >= (3,):
+    iscoroutinefunction: Callable[[Callable[..., Any]], bool] = ...  # really inspect.iscoroutinefunction
+    getfullargspec: Callable[[Any], FullArgSpec] = ...  # really inspect.getfullargspec
```

Comment:
You can use `from foo import bar as bar` to re-export a definition from another module. This avoids redefinition and you don't need to define `FullArgSpec` for Python 3.